v2/azure-arm.yaml

```diff
@@ -31,6 +31,7 @@ supportedResourcesReport:
   # resourcePath is a template used for generating a file path for checking whether docs for a resource have been generated
   # specified relative to the directory of outputPath, and supports the same placeholders
   resourcePathTemplate: "{group}/{version}.md"
+  currentRelease: v2.1.0
```

Comment:
Update docs/hugo/content/contributing/create-a-new-release.md documentation to mention we need to update this. Clarify when it should happen, before or after the actual release?

Reply:
I'll either do that, or update the workflow to update `currentRelease` automatically.